include/SimpleTextFloat.h

```diff
@@ -27,6 +27,7 @@
 #define SIMPLE_TEXT_FLOAT_H
 
 #include <QWidget>
+#include <QTimer>
```

Comment:
Because the `QTimer` class is used as a pointer type you can forward declare it in this header and move the actual include into the .cpp file.